satellite/metainfo/validation.go

```diff
@@ -244,13 +244,12 @@ func (endpoint *Endpoint) validatePointer(ctx context.Context, pointer *pb.Point
 
 	// TODO does it all?
```

Comment:
I know that it has not been added in this branch, but do you know if the question is still applying?

Reply:
We didn't have one story to verify and fill gaps in `pointer` struct validation. Lots of these rules were added with tasks only related to validation. I will remove this comment and make jira ticket to review metainfo validation.